command/agent/consul/check_watcher.go

```diff
@@ -22,8 +22,10 @@ type ChecksAPI interface {
 }
 
 // TaskRestarter allows the checkWatcher to restart tasks.
+//XXX just reference taskrunner/interfaces.TaskLifecycle directly? seems like
```

Comment:
I don't think it will be an import cycle. The goal of having a separate package for the interface is so everyone can import it without cycles